eth/downloader/peer.go

```diff
@@ -150,8 +150,8 @@ func (p *peerConnection) FetchHeaders(from uint64, count int) error {
 	p.headerStarted = time.Now()
 
 	// Issue the header retrieval request (absolute upwards without gaps)
-	go p.peer.RequestHeadersByNumber(from, count, 0, false)
 
+	go p.peer.RequestHeadersByNumber(from, count, 0, false)
```

Comment:
why not use gopool here

Reply:
It is used when clients doing fast sync and snap sync. Not related to performance